{
  "file": "test/error.js",
  "contents": "(The value of this property is over-sized, so it is validated independently.)",
  "attrs": {
    "description": "Fails by calling $ERROR (Strict Mode)",
    "negative": {
      "phase": "runtime",
      "type": "Test262Error"
    },
    "flags": {},
    "includes": [
      "assert.js"
    ]
  },
  "copyright": "",
  "scenario": "strict mode",
  "insertionIndex": 904,
  "raw": "/*---\ndescription: Fails by calling $ERROR\nnegative:\n  phase: runtime\n  type: Test262Error\n---*/\n\n$ERROR('failure message');\n"
}